core/templates/dev/head/components/question-directives/question-player/question-player.directive.ts

```diff
@@ -236,23 +238,32 @@ oppia.directive('questionPlayer', [
             return ctrl.totalQuestions;
           };
 
+          var isInPassOrFailMode = function() {
+            return (ctrl.questionPlayerConfig.questionPlayerMode &&
+              ctrl.questionPlayerConfig.questionPlayerMode.modeType ===
+              'PASS_FAIL');
```

Comment:
Maybe better to define this as a constant somewhere and use it throughout.

Reply:
Done.